### DISCUSSION TITLE
Error in the code on multiple devices

### DISCUSSION THREAD

**Discussion by sorz2122** [syssi/esphome-soyosource-gtn-virtual-meter#180] [General]
https://github.com/KinDR007/VictronMPPT-ESPHOME/blob/main/smartsolar-mppt-esp8266-example-multiple-uarts.yaml

Hello, there is an error in the configuration. In line 73, it currently says:

- platform: victron
  victron_id: victron1
  max_power_yesterday:
    name: "${device0} max power yesterday"

But it should be:

- platform: victron
  victron_id: victron1
  max_power_yesterday:
    name: "${device1} max power yesterday"

Liebe Grüße

**Comment by syssi** [comment]
This project is about the Soyosource inverter. Your issue looks like about the victron project.

**Reply by sorz2122** [reply]
That’s actually true, I’m sorry. I currently have both projects running and apparently mixed up the source. Anyway, a note for the charge controller.

**Reply by syssi** [reply]
I've pushed a fix. Thanks for your report! https://github.com/KinDR007/VictronMPPT-ESPHOME/pull/173
